moviepy/video/VideoClip.py

```diff
@@ -576,25 +577,40 @@ def blit_on(self, picture, t):
         on the given `picture`, the position of the clip being given
         by the clip's ``pos`` attribute. Meant for compositing.
         """
-        hf, wf = framesize = picture.shape[:2]
+        hf, wf = picture.size
 
-        if self.ismask and picture.max():
-            return np.minimum(1, picture + self.blit_on(np.zeros(framesize), t))
+        # ? I'm not so sure about the function of this code, need information
+        # if self.ismask and picture.max():
+        #     return np.minimum(1, picture + self.blit_on(np.zeros(framesize), t))
```

Comment:
do we need this?

Reply:
Thanks for your view, I annotated it since I have no idea about these code, but it still pass all tests anyway. Seems ok to delete them.